Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyViewModel.kt

```diff
@@ -119,7 +119,7 @@ class SurveyViewModel(
                 }
             val answer: Answer.Action? = question.answer as Answer.Action?
             if (answer != null && answer.result is SurveyActionResult.Date) {
-                val formatter = SimpleDateFormat("MMM dd, yyyy", Locale.ENGLISH)
+                val formatter = SimpleDateFormat("EEE, MMM d", Locale.ENGLISH)
```

Comment:
The data format pattern seems to be used multiple times, move it to a constant

Reply:
created a const for the simple date format pattern